README.md

```diff
@@ -54,6 +54,35 @@ jhipster --blueprints quarkus
 
 ```
 
+## Using Docker
+
+Download the Dockerfile:
+
+```bash
+mkdir docker
+cd docker
+wget https://github.com/jhipster/jhipster-quarkus/raw/master/docker/Dockerfile
+```
+
+Build the Docker images:
+
+```bash
+docker build -t jhipster-generator-quarkus:latest .
+```
+
+Make a folder where you want to generate the Service:
```

Comment:
What does the "service" term mean here ?

Reply:
app , which is going to generated by generator. Can suggest a generic word or name for it .